wgsl/index.bs

```diff
@@ -1781,6 +1781,26 @@ result vector is formed by operating on each component independently.
   </xmp>
 </div>
 
+WGSL also [=predeclared|predeclares=] the following [=type aliases=]:
+
+<table class='data'>
+  <thead>
+    <tr><th>Type<th>Predeclared alias<th>Restrictions
```

Comment:
Suggest swapping the first two columns.

I'd also suggest condensing  the number of rows by leaving _N_ unspecified. We don't list vec2, vec3, vec4 individually above. Then add an example, which is probably deserved anyways.

Reply:
> I'd also suggest condensing the number of rows by leaving N unspecified. 

Both @dj2 and @jimblandy like the explicitness.  I like the explicitness because it makes it easier to search and find.